src/core/pages/signin/Authenticate.vue

```diff
@@ -22,7 +22,10 @@
         <router-link class="row justify-end" :to="{ name: 'forgotPassword' }">
           <small>Mot de passe oublié ?</small>
         </router-link>
-        <div class="row justify-center">
+        <div class="row justify-center first-login">
+          <router-link class="q-mr-lg" :to="{ name: 'forgotPassword' }">
+            <small>Première connexion ?</small>
+          </router-link>
```

Comment:
Est-ce que tu peux demander a romain la taille de la police ? J'ai l'impression que c'est plus grand que mot de passe oublié. 
Aussi il avait noté "C'est ma première connexion"

Reply:
plutot que de passer par un router-link, on peut utiliser le `<ni-button...` 
Ssi tu reprends le meme composant que pour "Connexion" mais en enlevant la props `:flat="false"`, ca inverse les couleur et donne ce qu'on veut. ca evitera d'avoir des souci de taille et alignement